modules/platform/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -111,47 +118,46 @@ public DocumentModel createOrUpdate(FileImporterContext context) throws IOExcept
 
             ZipEntry index = zip.getEntry(MARKER);
             try (Reader reader = new InputStreamReader(zip.getInputStream(index));
-                 CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
 
                 Map<String, Integer> header = csvParser.getHeaderMap();
                 for (CSVRecord csvRecord : csvParser) {
                     String type = null;
-                    String id = null;
+                    String name = null;
                     Map<String, String> stringValues = new HashMap<>();
                     for (String headerValue : header.keySet()) {
                         String lineValue = csvRecord.get(headerValue);
                         if ("type".equalsIgnoreCase(headerValue)) {
                             type = lineValue;
-                        } else if ("id".equalsIgnoreCase(headerValue)) {
-                            id = lineValue;
+                        } else if ("name".equalsIgnoreCase(headerValue)) {
+                            name = lineValue;
                         } else {
                             stringValues.put(headerValue, lineValue);
                         }
                     }
 
+                    if (StringUtils.isBlank(name)) {
+                        log.error("Can not create or update doc without a name, skipping line: {}", name);
```

Comment:
I would use a `log.warn` here, ERROR seems a bit strong for something about which the server admins can't do anything. Also logging the name is useless as it's blank, maybe log `csvRecord.getRecordNumber()` or `csvRecord.getCharacterPosition()` instead?

Reply:
sorry I missed that @efge , I made an update for this